docs/dev/install.rst

```diff
@@ -114,7 +114,9 @@ save some work while typing docker compose commands. This section explains these
     * ``--init`` is used the first time this command is ran to run initial migrations, create an admin user, etc
     * ``--no-reload`` makes all celery processes and django runserver
       to use no reload and do not watch for files changes
-    * ``--ngrok`` is useful when it's required to access the local instance from outside (e.g. GitHub webhook)
+    * ``--no-django-debug`` runs all containers with ``DEBUG=False``
+    * ``--http-domain`` configures an external domain for the environment (useful for Ngrok or other https proxy)
```

Comment:
Doesn't this interfere with the `--ngrok` option from https://github.com/readthedocs/common/pull/151?

Reply:
Ah, you removed that option... I'm still sleepy it seems. I'll need to come back to this when I'm 100% awake 😴